activesupport/lib/active_support/log_subscriber.rb

```diff
@@ -47,8 +47,8 @@ module ActiveSupport
   #         if exception
   #           exception_object = event.payload[:exception_object]
   #
-  #           "[ERROR] #{event.payload[:name]}: #{exception.join(', ')} " \
-  #           "(#{exception_object.backtrace.first})"
+  #           info "[ERROR] #{event.payload[:name]}: #{exception.join(', ')} " \
```

Comment:
What about setting this example as `error` instead of `info`?

Reply:
Makes sense. I'll change it 👍